doc/api/next_api_changes/removals/22465-AL.rst

```diff
@@ -0,0 +1,4 @@
+``mpl_toolkits.axes_grid1.axes_size.AddList``
```

Comment:
Should this have ended up in deprecations rather than removals?

Reply:
Ah yes, I think so.